pkg/restore.go

```diff
@@ -311,9 +312,8 @@ func (opt *mongoOptions) restoreMongoDB(targetRef api_v1beta1.TargetRef) (*resti
 			Host:       hostKey,
 			SourceHost: hostKey,
 			FileName:   opt.defaultDumpOptions.FileName,
-			Snapshot:   opt.defaultDumpOptions.Snapshot,
+			Snapshot:   opt.getSnapshotForHost(hostKey, restoreSession.Spec.Target.Rules),
```

Comment:
For configServer, the hostKey is set to [confighost](https://github.com/stashed/mongodb/blob/master/pkg/utils.go#L42).  That means, in restoresession.spec.rules.targetHosts, users has to set it `confighost`. But in appbinding.spec.parameters , that is `configServer`.  

I think, this type of different naming is confusing.  May be, renaming this const as `configServer` ?

Reply:
Yes. But, this will break the backward compatibility. Existing snapshots have been created with `confighost`